channeldb/migrations_test.go

```diff
@@ -564,3 +564,123 @@ func TestMigrateGossipMessageStoreKeys(t *testing.T) {
 		migrateGossipMessageStoreKeys, false,
 	)
 }
+
+// TestOutgoingPaymentsMigration checks that OutgoingPayments are migrated to a
+// new bucket structure after the migration.
+func TestOutgoingPaymentsMigration(t *testing.T) {
```

Comment:
should this be added in the same commit as the migration?

Reply:
Prefer splitting it do avoid adding migration+fetch payment logic+test in one big commit.